packages/common/db/baseConfig.ts

```diff
@@ -0,0 +1,19 @@
+interface DatabaseConfig {
+  host: string
+  user: string
+  password: string
+  database: string
+  port: number
+  ssl: boolean
+  schema: string
+}
+
+export const baseConfig: DatabaseConfig = {
+  host: process.env.DB_HOST ?? process.env.DB_AUTH_HOST ?? "localhost",
+  user: process.env.DB_USER ?? process.env.DB_AUTH_USER ?? "bichard",
+  password: process.env.DB_PASSWORD ?? process.env.DB_AUTH_PASSWORD ?? "password",
+  database: process.env.DB_DATABASE ?? process.env.DB_AUTH_DATABASE ?? "bichard",
+  port: parseInt(process.env.DB_PORT ?? process.env.DB_AUTH_PORT ?? "5432", 10),
```

Comment:
```suggestion
  port: Number(process.env.DB_PORT ?? process.env.DB_AUTH_PORT ?? "5432"),
```

So we don't have to specify the base